app/products/calls/websocket.ts

```diff
@@ -7,13 +7,14 @@ import {encode} from '@msgpack/msgpack/dist';
 
 export default class WebSocketClient extends EventEmitter {
     private ws: WebSocket | null;
-    private seqNo = 0;
+    private seqNo = 1;
     private connID = '';
     private eventPrefix = `custom_${Calls.PluginId}`;
 
-    constructor(connURL: string) {
+    constructor(connURL: string, authToken: string) {
         super();
-        this.ws = new WebSocket(connURL);
+
+        this.ws = new WebSocket(connURL, [], {headers: {authorization: `Bearer ${authToken}`}});
```

Comment:
I tried to debug this a bit but didn't get a complete understanding. For some reason the mobile client isn't sending the cookies correctly when I have a MM served from a subdirectory so I am setting the header which works fine. I could have used the auth_challenge event as well but we avoid one request this way.

Reply:
Probably the path for the cookie is setting at root and not at the subpath thus not finding it... anyway, this approach should solve it for all